src/arfs/arfsdao.ts

```diff
@@ -214,6 +221,8 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		super(arweave, undefined, undefined, caches);
 	}
 
+	private shouldProgressLog = process.env['ARDRIVE_PROGRESS_LOG'] === '1';
```

Comment:
Strictly equal to `'1'` VS not empty :opionion:
```suggestion
	private shouldProgressLog = process.env['ARDRIVE_PROGRESS_LOG'] !== '';
```
_Non-blocking comment_

Reply:
This is following the same pattern we used for cache logging. I don't like the "not empty" check because it means the following might not work as expected:
```
export ARDRIVE_PROGRESS_LOG=false
export ARDRIVE_PROGRESS_LOG=0
export ARDRIVE_PROGRESS_LOG=no
etc.
```